python/ray/serve/_private/http_proxy.py

```diff
@@ -535,19 +449,191 @@ async def __call__(self, scope, receive, send):
                 tags={
                     "route": route_path,
                     "error_code": status_code,
-                    "method": scope["method"].upper(),
+                    "method": method,
                 }
             )
             self.deployment_request_error_counter.inc(
                 tags={
                     "deployment": handle.deployment_name,
                     "error_code": status_code,
-                    "method": scope["method"].upper(),
+                    "method": method,
                     "route": route_path,
                     "application": app_name,
                 }
             )
 
+    async def send_request_to_replica_unary(
```

Comment:
To confirm, this code is the exact same as `_send_request_to_handle`, correct?

Reply:
yup this is unchanged